pyccel/ast/cwrapper.py

```diff
@@ -119,3 +119,58 @@ def flags(self):
     @property
     def args(self):
         return self._result_args
+
+class CastFunction(FunctionDef):
+    """Represents a cast function definition."""
+
+    def __init__(self,
+        name,
+        cast_type,
+        arguments,
+        ret,
+        results):
+        self._name = name
+        self._arguments = arguments
+        self._ret = ret
+        self._results = results
+        self._cast_type = cast_type
+        body_ = ''
+        #TODO I dont know if the build of body shoudl be done here or in the cwrapper
+        #TODO this is just a tmp way of printing this shoudlbe improved later
+        if self._cast_type == 'pyint_to_bool':
+            body_ += '{} = {} != 0;\n'.format(self.results[0].name, self._arguments[0].name)
```

Comment:
The printing should avoid code duplication. It should therefore make use of the other functions eg:
`Assign(self.results[0], Bool(self._arguments[0]))`
(although I assume that this is what your comment refers to)

Reply:
yes thats true